client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -1,152 +1,217 @@
-				{ this.props.translate( 'Upgrade' ) }
-			</Button>
-		);
-	}
+		return version_compare( wpVersion, pluginTestedVersion, '>' );
+	} );
 
-	renderPlaceholder() {
-		/* eslint-disable wpcalypso/jsx-classname-namespace */
-		return (
-			<li className="plugins-browser-item is-placeholder">
-				<span className="plugins-browser-item__link">
-					<div className="plugins-browser-item__info">
-						<PluginIcon size={ this.props.iconSize } isPlaceholder={ true } />
-						<div className="plugins-browser-item__title">…</div>
-						<div className="plugins-browser-item__author">…</div>
-						<div className="plugins-browser-item__description">…</div>
-					</div>
-				</span>
-			</li>
-		);
-		/* eslint-enable wpcalypso/jsx-classname-namespace */
+	if ( isPlaceholder ) {
+		return <Placeholder iconSize={ iconSize } />;
 	}
 
-	render() {
-		const { isPlaceholder, plugin, iconSize, variant, translate } = this.props;
-		if ( isPlaceholder ) {
-			return this.renderPlaceholder();
-		}
-
-		const classNames = classnames( 'plugins-browser-item', variant );
-
-		return (
-			<li className={ classNames }>
-				<a
-					href={ this.getPluginLink() }
-					className="plugins-browser-item__link"
-					onClick={ this.trackPluginLinkClick }
-				>
-					<div className="plugins-browser-item__info">
-						<PluginIcon size={ iconSize } image={ plugin.icon } isPlaceholder={ isPlaceholder } />
-						<div className="plugins-browser-item__title">{ plugin.name }</div>
-						{ variant === PluginsBrowserElementVariant.Extended && (
+	const classNames = classnames( 'plugins-browser-item', variant );
+	return (
+		<li className={ classNames }>
+			<a
+				href={ pluginLink }
+				className="plugins-browser-item__link"
+				onClick={ trackPluginLinkClick }
+			>
+				<div className="plugins-browser-item__info">
+					<PluginIcon size={ iconSize } image={ plugin.icon } isPlaceholder={ isPlaceholder } />
+					<div className="plugins-browser-item__title">{ plugin.name }</div>
+					{ variant === PluginsBrowserElementVariant.Extended && (
+						<>
 							<div className="plugins-browser-item__author">
 								{ translate( 'by ' ) }
 								<span className="plugins-browser-item__author-name">{ plugin.author_name }</span>
 							</div>
+							{ dateFromNow && (
+								<div className="plugins-browser-item__last-updated">
+									{ translate( 'Last updated ' ) }
+									<span className="plugins-browser-item__last-updated-value">{ dateFromNow }</span>
+								</div>
+							) }
+						</>
+					) }
+					<div className="plugins-browser-item__description">{ plugin.short_description }</div>
+				</div>
+				{ isUntestedVersion && (
+					<div className="plugins-browser-item__untested-notice">
+						<Icon size={ 20 } icon={ info } />
+						<span className="plugins-browser-item__untested-text">
+							{ translate( 'Untested with your version of WordPress' ) }
+						</span>
+					</div>
```

Comment:
Have you considered reusing the `notice` component?
https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/components/notice/index.jsx#L29

Reply:
Tried it quickly here, but it seems very different from what we have on the mock.

<img width="476" alt="Screen Shot 2021-12-08 at 15 56 00" src="https://user-images.githubusercontent.com/5039531/145275343-eaed94f7-29a8-4e79-b03b-f303950b0908.png">

Reply:
Cool, it seems that the code added is less than the code that we would have to override.